packages/pagination/README.md

```diff
@@ -14,6 +14,8 @@ npm install react react-dom prop-types styled-components @zendeskgarden/react-th
 
 ## Usage
 
+### Offset Pagination
+
 ```jsx static
 import { ThemeProvider } from '@zendeskgarden/react-theming';
 import { Pagination } from '@zendeskgarden/react-pagination';
```

Comment:
I know it's a larger refactor, but if we're going to commit to this naming for this component, shouldn't it be <OffsetPagination> for the component name (instead of <Pagination>)? This will stop us from having the same effect we have for Notifications where we have a Notification component inside the Notifications package and it can be confusing to tell what you're referring to.

Reply:
I originally had renamed this to `OffsetPagination` because I think the updated name helps clarify the `react-pagination` package. It makes it explicit that there two patterns to choose from depending on the use case.

Unfortunately renaming `Patination` to `OffsetPagination` would result in a breaking change that would require a major version bump — which isn't planned for the upcoming releases. I really like the idea of the rename though, and would love to suggest discussing it for the next major version.

Reply:
Ah, interesting. Do we have a running list of planned major version changes?

Reply:
I'm actually quite interested in this workflow. It seems a real shame to indefinitely punt refactor changes like this when they play a rather important role in how everyone learns to talk about the components. If it's a necessity than it is what it is but maybe we start a Jira epic?

Reply:
@rossmoody we'll track and commit to this in a staggered manner. Something like first deprecating `Pagination` and providing a non-deprecated `OffsetPagination` alias. And then subsequently removing `Pagination` in a future, major v9.

Reply:
Niceeeeeeeee